### ISSUE TITLE
Support sails.socket.join with redis

### ISSUE THREAD

**Issue by luislobo** [balderdashy/sails#2139]
I was reading the code, to learn how Room Join it is implemented in Sails, as we are having some issues using them, and read a TODO Comment in the .join function:
// TODO: add support for optional callback (for use w/ redis)

We were planning to use redis, as the socket repository, since we will need to have several node servers.

Does the current implementation supports it?

Thanks!

**Comment by CWyrtzen** [comment]
@luislobo is this still an issue?

**Comment by sailsbot** [comment]
Thanks for posting, @luislobo.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
